ai-platform-unified/notebooks/unofficial/model_monitoring/model_monitoring.ipynb

```diff
@@ -0,0 +1,1006 @@
+{
```

Comment:
<p>It's possible to use the Vertex AI sdk (preferred) to do this instead of <code>gcloud.</code> </p><p>It's up to you.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/318/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>I kind of like having a combination of the sdk, gcloud, and the console in this lab so I'm ok with command line here.</p>